src/Source/Extras.swift

```diff
@@ -291,13 +291,17 @@ public extension String
     public func snakeCaseToCamelCase() -> String {
         let items = self.components(separatedBy: "_")
         var camelCase = ""
-        items.enumerated().forEach {
-            if ($1.isInteger()) {
+        for (i, x) in items.enumerated() {
+            if i == 0 {
```

Comment:
Do you want to add x twice for the first letter? Once lower and once upper? I could be missing the point here, but calling it out just to be safe..

Reply:
"Continue" breaks so I don't think it's added twice? We can peep this together f2f and walk through the logic to make sure.

Reply:
makes sense, thx! :) 👍

Reply:
Hm, this is actually code that I pulled in from the original project. I actually think it's working fine like this. The i==0 is just the special case where we don't want to x.capitalize() the first character, we want to keep it lower.

Reply:
👍